### PR TITLE
add Overview/Taxonomy

### PR DESCRIPTION
**Pull request by mirzak**
Based on work done in https://github.com/mendersoftware/mender-docs/blob/documentation-restructuring/02.Overview/12.Taxonomy/docs.md and #1024

### CODE REVIEW

**Comment by mirzak** [comment]
@drewmoseley @eysteins @merlin-northern, a final approval?

**Comment by estenberg** [comment]
A few comments:

* Sort alphabetically (maybe I missed some context?)
* Try to explain the purpose (why) of each, where it makes sense, not only "what".
* It feels in general a bit too detailed, as opposed to focus on the concept, or what it is used for.
* Keep this to "objects", not "actions. For example I see no point in "Accepting a device" in the taxonomy when we have "Accepted device". We should only have "Accepted device".


For example, as opposed to:

* Device ID: "A string of letters, numbers, and dashes, uniquely identifying a device in Mender."

say

* Device ID: A unique string that identifies a device in the Mender server so devices can easily be referenced in APIs."

**Comment by mirzak** [comment]
@drewmoseley @estenberg final review?

File: 02.Overview/15.Taxonomy/docs.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+* _Artifact Name_ - A human-readable string uniquely describing an Artifact,
+used by UI and API for identification purposes.
+
+* _Artifact Payload_ - Consists of a type and actual data to be installed on a
+device. It is stored inside a Mender Artifact. It could be a rootfs image,
+package, container, or other. See the documentation on
+[Artifact](../05.Artifact/docs.md) for more information.
+
+* _Authentication Set_ - A combination of an identity and public key for a
+device, determining if a given device is allowed to check for- and apply
```
Comment (merlin-northern) [inline 02.Overview/15.Taxonomy/docs.md:26]: passive :)

File: 02.Overview/15.Taxonomy/docs.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+* _Board integration_ - The low-level integration required to enable system
+updates with Mender on a board. Often includes OS boot loader and storage
+integration and requires customization based on the hardware and OS.
+
+* _Deployment_ - The process of delivering software to devices. It consists of
+at least a group of devices and an Artifact name.
+
+* _Device_ - A single unit that is able to connect to the Mender server, usually
+an independent product. It is represented on the server by its identity and
```
Comment (merlin-northern) [inline 02.Overview/15.Taxonomy/docs.md:46]: passive

File: 02.Overview/15.Taxonomy/docs.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+* _Mender Hub Integration(s)_ - A contribution on
+[Mender Hub](https://hub.mender.io/c/board-integrations?target=_blank)
+of a Mender Board integration for a specific board.
+
+* _Mender Server_ - An application implementing Mender API, and the web UI,
+providing updates to devices.
+
+* _Organization_ - A single customer environment in the Mender server. Also
+known as a Tenant. Note that multi-tenancy is only supported in Mender
```
Comment (merlin-northern) [inline 02.Overview/15.Taxonomy/docs.md:65]: passive

File: 02.Overview/15.Taxonomy/docs.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+* _Artifact_ -  An archive containing everything needed for an update of a
+device, including the Artifact Payload itself and metadata such as signatures.
+See the documentation on [Artifact](../05.Artifact/docs.md) for more information.
+
+* _Artifact Name_ - A human-readable string uniquely describing an Artifact,
+used by UI and API for identification purposes.
+
+* _Artifact Payload_ - Consists of a type and actual data to be installed on a
+device. It is stored inside a Mender Artifact. It could be a rootfs image,
```
Comment (merlin-northern) [inline 02.Overview/15.Taxonomy/docs.md:21]: passive

**Comment by mirzak** [comment]
I was actually aware of the passive voice but did not feel the itch to resolve it in this section specifically. 

These are reported:

```
$ bash scripts/passive.sh 02.Overview/15.Taxonomy/docs.md 
02.Overview/15.Taxonomy/docs.md:20:be installed
02.Overview/15.Taxonomy/docs.md:21:is stored
02.Overview/15.Taxonomy/docs.md:26:is allowed
02.Overview/15.Taxonomy/docs.md:46:is represented
02.Overview/15.Taxonomy/docs.md:69:being authorized
02.Overview/15.Taxonomy/docs.md:80:be forbidden
```

But if you insist :)

**Comment by merlin-northern** [comment]
> I was actually aware of the passive voice but did not feel the itch to resolve it in this section specifically.
> 
> These are reported:
> 
> ```
> $ bash scripts/passive.sh 02.Overview/15.Taxonomy/docs.md 
> 02.Overview/15.Taxonomy/docs.md:20:be installed
> 02.Overview/15.Taxonomy/docs.md:21:is stored
> 02.Overview/15.Taxonomy/docs.md:26:is allowed
> 02.Overview/15.Taxonomy/docs.md:46:is represented
> 02.Overview/15.Taxonomy/docs.md:69:being authorized
> 02.Overview/15.Taxonomy/docs.md:80:be forbidden
> ```
> 
> But if you insist :)

have I ever been seen insisting?

**Comment by mirzak** [comment]
> looks good apart from lots of passive voice, which I was thought not to use at all.

I agree with you, and will fix.

**Pull request merged**